.github/workflows/dependencies-head.yml

```diff
@@ -91,9 +91,9 @@ jobs:
         python -m pip list
     - name: Test with pytest
       run: |
-        pytest --ignore tests/benchmarks/ --ignore tests/contrib --ignore tests/test_notebooks.py
+        pytest --ignore tests/contrib --ignore tests/benchmarks --ignore tests/test_notebooks.py
 
-  uproot4:
+  uproot5:
```

Comment:
should be a separate PR

Reply:
I can do it if you want, but this is a cosmetic change in that https://github.com/scikit-hep/uproot4 resolves to https://github.com/scikit-hep/uproot5.